packages/seacas/libraries/ioss/src/Ioss_Utils.C

```diff
@@ -2436,13 +2445,16 @@ namespace {
       // 'connectivity' field, but it is only interesting on the
       // Ioss::ElementBlock class. On the other classes, it just
       // generates overhead...
+
       if (field_name == "connectivity" && ige->type() != Ioss::ELEMENTBLOCK) {
         continue;
       }
       if (field_name == "ids") {
         continue;
       }
-      if (Ioss::Utils::substr_equal(prefix, field_name)) {
+
```

Comment:
Is the `Ioss::Utils::substr_equal()` not valid here?

Reply:
This is a cherry pick merge error.  I fixed it in 135650d.